### PR TITLE
feat: Add Erlang and Go libraries

### PR DESCRIPTION
**Pull request by moritzploss-k**
## About

Add Erlang and Go libraries to simplify working with API errors.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -5,3 +5,10 @@ build:
```
Comment (RSBRK) [inline Makefile:5]: Nitpick: Different indentation

File: Makefile [hunk 2]
```diff
@@ -5,3 +5,10 @@ build:
 
 lint: build
 	@docker run --rm $(PROJECT_NAME)
+
+erl:
+	rebar3 dialyzer;
+	rebar3 eunit
+
+go:
```
Comment (RSBRK) [inline Makefile:13]: Maybe call these "test-erl" and "test-go"

File: erlang/kivra_api_errors.app.src [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  {vsn, git},
+  {registered, []},
+  {applications,
+   [kernel,
+    stdlib,
+    jsx
+   ]},
+  {env,[]},
+  {modules, []},
+  {licenses, ["MIT"]},
```
Comment (RSBRK) [inline erlang/kivra_api_errors.app.src:12]: Do we have some internal guidelines for which license to use?

File: rebar.lock [hunk 4]
```diff
@@ -0,0 +1,8 @@
+[{<<"jiffy">>,
+  {git,"git@github.com:kivra/jiffy.git",
+       {ref,"9ea1b35b6e60ba21dfd4adbd18e7916a831fd7d4"}},
+  0},
+ {<<"jsx">>,
```
Comment (RSBRK) [inline rebar.lock:5]: Is this used? I didn't see it anywhere.

**Review by RSBRK — COMMENTED** [review]
Can we remove `go.sum` as it is empty or is it required anyway?

File: rebar.lock (hunk 4 above)
Comment (moritzploss-k) [reply rebar.lock:5]: no, I was changing this back and forth, will remove it

**Comment by moritzploss-k** [comment]
> Can we remove `go.sum` as it is empty or is it required anyway?

we can remove it

File: erlang/kivra_api_errors.app.src (hunk 3 above)
Comment (moritzploss-k) [reply erlang/kivra_api_errors.app.src:12]: I think we use MIT in most places

File: Makefile (hunk 1 above)
Comment (moritzploss-k) [reply Makefile:5]: not sure I get 😄 Compared to what?

Comment (RSBRK) [reply Makefile:5]: It's on the line above, on row 4.

Comment (RSBRK) [reply Makefile:5]: It's just Github showing this strangely

**Review by RSBRK — APPROVED** [review]

**Pull request merged**